vision_agent_tools/shared_types.py

```diff
@@ -124,3 +124,76 @@ class Config:
                 }
             }
         }
+
+
+# florence2
+
+
+class PromptTask(str, Enum):
+    """Valid task_prompts options for the Florence2 model."""
+
+    CAPTION = "<CAPTION>"
+    CAPTION_TO_PHRASE_GROUNDING = "<CAPTION_TO_PHRASE_GROUNDING>"
+    DETAILED_CAPTION = "<DETAILED_CAPTION>"
+    MORE_DETAILED_CAPTION = "<MORE_DETAILED_CAPTION>"
+    DENSE_REGION_CAPTION = "<DENSE_REGION_CAPTION>"
+    OPEN_VOCABULARY_DETECTION = "<OPEN_VOCABULARY_DETECTION>"
+    OBJECT_DETECTION = "<OD>"
+    OCR = "<OCR>"
+    OCR_WITH_REGION = "<OCR_WITH_REGION>"
+    REGION_PROPOSAL = "<REGION_PROPOSAL>"
+    REFERRING_EXPRESSION_SEGMENTATION = "<REFERRING_EXPRESSION_SEGMENTATION>"
+    REGION_TO_SEGMENTATION = "<REGION_TO_SEGMENTATION>"
+    REGION_TO_CATEGORY = "<REGION_TO_CATEGORY>"
+    REGION_TO_DESCRIPTION = "<REGION_TO_DESCRIPTION>"
+
+
+class Florence2ModelName(str, Enum):
+    FLORENCE_2_LARGE = "microsoft/Florence-2-large"
+    FLORENCE_2_BASE_FT = "microsoft/Florence-2-base-ft"
+
+
+class Florence2TextResponse(BaseModel):
+    text: str
+
+
+class ODResponse(BaseModel):
+    labels: list[str]
+    bboxes: list[list[float]]
+
+
+class Florence2OCRResponse(BaseModel):
+    labels: list[str]
+    quad_boxes: list[list[float]]
+
+
+class Florence2SegmentationResponse(BaseModel):
+    labels: list[str]
+    polygons: list[list[list[float]]]
+
+
+class Florence2OpenVocabularyResponse(BaseModel):
+    bboxes: list[list[float]]
+    bboxes_labels: list[str]
```

Comment:
should we cast it to `labels` to keep consistency ? 
```suggestion
    bboxes_labels: list[str] = Field(alias="labels")
```

Reply:
I think is better to keep the way florence2 sends, to not get confused by the polygon labels